bin/docker-entrypoint.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+# Wrapper script to combine Java options at runtime
+# This allows USER_OPTS_APPEND to be set at runtime while preserving
+# the Quarkus and internal options defined at build time.
+
+# Combine all OPTS variables into JAVA_OPTS_APPEND at runtime
+# This ensures runtime values for USER_OPTS_APPEND are included
+export JAVA_OPTS_APPEND="${QUARKUS_OPTS_APPEND:-} ${INTERNAL_OPTS_APPEND:-} ${USER_OPTS_APPEND:-}"
+if [ -z "${JAVA_OPTS_APPEND:-}" ]; then
+  export JAVA_OPTS_APPEND=$(echo "${QUARKUS_OPTS_APPEND:-} ${INTERNAL_OPTS_APPEND:-} ${USER_OPTS_APPEND:-}" | xargs)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)